src/ActionInfoPanel.kt

```diff
@@ -180,13 +157,14 @@ class ActionInfoPanel(project: Project, textFragments: List<Pair<String, Font?>>
     private fun List<Pair<String, Font?>>.mergeFragments() : List<Pair<String, Font?>> {
         var result = ArrayList<Pair<String, Font?>>()
         for (item in this) {
-            val last = result.last
-            if (last != null && last.second == item.second) {
-                result.remove(result.lastIndex)
-                result.add(Pair(last.first + item.first, last.second))
-            }
-            else {
+            if(result.isEmpty()){
```

Comment:
result.last() throws without this check (may be we can check it in the better way)